cats/__init__.py

```diff
@@ -69,6 +69,15 @@ def parse_arguments():
         prog="cats", description=description_text, epilog=example_text
     )
 
+    def positive_integer(string):
+        try:
+            n = int(string)
+            assert n >= 0
+            return n
+        except (ValueError, AssertionError):
+            logging.error("Must be a positive integer")
+            sys.exit(1)
```

Comment:
raising a ValueError should be sufficient for the parser to error out

Reply:
That's a very good catch, I just stole that from your PR. I'm still keeping the `assert n >= 0` there no?

Reply:
```python
def positive_integer(string):
        n = int(string)
        assert n >= 0
        return n
```